src/transformers/generation/flax_utils.py

```diff
@@ -330,7 +330,6 @@ def generate(
 
         generation_config = copy.deepcopy(generation_config)
         model_kwargs = generation_config.update(**kwargs)  # All unused kwargs must be model kwargs
-        generation_config.validate()
```

Comment:
Because I've added the `.validate()` call to `.update()`, which is the line that precedes all these removals :) It makes more sense to validate at update time, rather than relying on the user to manually validate.